src/utils.cc

```diff
@@ -490,6 +492,11 @@ std::string GetDefaultResourceDirectory() {
 
   std::string resource_directory =
       std::string(ENSURE_STRING_MACRO_ARGUMENT(DEFAULT_RESOURCE_DIRECTORY));
+  if (resource_directory.size() >= 2 && resource_directory[0] == '"' &&
```

Comment:
Add a comment, ie,

`// Remove double quoted resource dir if it was passed with quotes by the build system.`